src/main/java/duke/task/TaskManager.java

```diff
@@ -0,0 +1,174 @@
+package duke.task;
+
+import duke.exception.TodoFormatException;
+import duke.exception.DeadlineFormatException;
+import duke.exception.EventFormatException;
+import duke.exception.EmptyTasklistException;
+import duke.exception.DoneFormatException;
+import duke.exception.InvalidTaskIdException;
+import duke.exception.TaskAlreadyDoneException;
+import duke.exception.DeleteFormatException;
+
+import duke.ui.DukeInterface;
+
+import java.util.ArrayList;
+
+public class TaskManager {
+
+    public ArrayList<Task> tasks;
+    private DukeInterface dukeUI;
+
+    private final String ADD_TASK_MSG = "Chomp-chomp! I've added this new task [\uD83D\uDCDD]:";
+    private final String SET_TASK_COMPLETE_MSG = "Burrrp! I've marked this task as done [\u2705]:";
+    private final String DELETE_TASK_MSG = "Blaargh! I've deleted this task from the list [\uD83D\uDCDD]:";
+    private final String PRINT_TASKLIST_MSG = "Ahh! Here are the tasks in your list [\uD83D\uDCC5]:";
+
+    private final String TODO_FORMAT_REGEX = "";
+    private final String DEADLINE_FORMAT_REGEX = ".+/by.+";
+    private final String EVENT_FORMAT_REGEX = ".+/at.+";
+    private final String DONE_FORMAT_REGEX = "\\d+";
+    private final String DELETE_FORMAT_REGEX = "\\d+";
+
+    public TaskManager() {
+        tasks = new ArrayList<Task>();
+        dukeUI = new DukeInterface();
+    }
+
+    public void addToDo(String todoInfo) throws TodoFormatException {
```

Comment:
`todo` are to separate words so follow camelCase variable name: `toDoInfo`. Similar comment for the exception naming (follow PascalCase)